news/pytest-warnings-others.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* No news added
```

Comment:
please use something like:
```
* no news added: intercepting warnings generated by tests on purpose not needed by users or devs.
```
With that said, I think this probably would be good to hvae a news.....

Reply:
Yup noted. There are other remaining warnings, etc. I will include that news item